test/src/e2e_vm_tests/test_programs/should_pass/stdlib/u128_div_test/src/main.sw

```diff
@@ -0,0 +1,25 @@
+script;
+
+use std::assert::assert;
+use std::result::*;
+use std::u128::*;
```

Comment:
The `u128` coming from the `std` just looks wrong. I think the additional `types` (like `B512` etc.) should be moved somewhere into `core::`

Reply:
I'm not sure if they belong in `core`, but we did think about factoring them out into `std::types` or something like that. There will be more and more of these abstractions and it would be good to organise them. That might be better handled in a separate PR though.

Reply:
agree that we could create a subdirectory in `std` to collect these types. There's already an open issue about reorganizing std where I belive this is discussed, will take a look.
`core` is for language-defined primitive types, this fits in std nicely.

Reply:
`core`'s only reason for existence is to define operators on primitive types, so that `std` can use them.